frontend/src/pages/DashboardPage.tsx

```diff
@@ -1,18 +1,23 @@
 import "./DashboardPage.css";
+import {useNavigate} from "react-router-dom";
+import Accordion from "../components/Accordion";
+import useGetTripsQuery from "../hooks/useGetTripsQuery";
 
 export default function DashboardPage() {
     // if not logged in, then navigate.to LoginPage?
 
-    // get all trips from API
-    // if no trips yet, navigate.to TripFormPage?
+    const navigate = useNavigate();
```

Comment:
Since we are all going to end up using the navigate, I'll put it in App.tsx and we can merge it in there

Reply:
Wie meinst du? Man muss das doch da holen wo mans braucht? bin nich ganz sicher, ob ich dich richtig verstanden hab, aber wenn wir das nur in App.tsx reinziehen, dann müssten wir es ja durch alle components als props durchreichen. Fühlt sich irgendwie nicht richtig an. Was meinst du?